src/Roslyn.Suppressions/Roslyn.Suppressions/SettingsFile/RoslynSettingsFileInfo.cs

```diff
@@ -28,12 +28,13 @@ internal static class RoslynSettingsFileInfo
         public static readonly string Directory = PathHelper.GetTempDirForTask(false, "Roslyn");
 
         /// <summary>
-        /// Returns the full file path for the given project key
+        /// Returns the full file path for the given solution name
```

Comment:
No code changes: I just updated the variable names and comments to make it clearer that we're working with a solution name, not a Sonar project key.